azure-iot-hub-devicesdk/samples/send_telemetry_device_sync.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is OK for now, but Pierre floated the idea of having a single uber-sample for sync code.  We should close on this.  If we decide to stick with multiple sync samples, maybe they should go into a different location (like samples/sync or samples_sync)

Reply:
I agree that this sample is kind of unnecessary since we don't want tons of sync samples. However I didn't want to remove anything that already existed, I figured removals could be a separate PR.

---
In reply to: [246912675](https://github.com/Azure/azure-iot-sdk-python-preview/pull/39#discussion_r246912675) [](ancestors = 246912675)